lib/mulder.rb

```diff
@@ -9,7 +9,7 @@ module Mulder
   require_relative 'mulder/version'
 
   def self.search(app, env, role, format, config_file)
-    config     = Mulder::Config.from(:yaml, file: config_file)
+    config     = Isomer::Configuration.hydrate(Mulder::CONFIG, Isomer::Sources::Yaml.new(config_file))
```

Comment:
Here I could also add in an `Environment` source if we wanted to be able to load in via `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` as well as the yaml file